recipes/cmake/binary/conandata.yml

```diff
@@ -0,0 +1,103 @@
+sources:
+  "3.20.6":
```

Comment:
Maybe it's a good opportunity to begin following CCI convention regarding version sort: https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/prebuilt_tool_package/config.yml#L2